salem/sio.py

```diff
@@ -1036,8 +1117,7 @@ def is_rotated_proj_working():
                       atol=1e-5).all()
 
 
-def open_metum_dataset(file, pole_longitude=None, pole_latitude=None,
-                       central_rotated_longitude=0., **kwargs):
+def open_metum_dataset(file, pole_longitude=None, pole_latitude=None, central_rotated_longitude=None, automatic_proj = False, **kwargs):
```

Comment:
I think I would prefer to have an "open_cosmo_dataset" function instead. It is more explicit (MetUM is [another model](https://www.metoffice.gov.uk/research/approach/modelling-systems/unified-model/index))